extensions/wikia/StoriesLinkTag/StoriesLinkTagHelper.class.php

```diff
@@ -8,6 +8,7 @@ class StoriesLinkTagHelper extends WikiaModel {
 	const CACHE_TTL = 3600;
 	const FANDOM_API_URL = 'http://fandom.wikia.com/wp-json/wp/v2/';
 	const VALID_HOST = 'fandom.wikia.com';
+	const VAR_NAME_API_TIMEOUT = 'wgStoriesLinkTagTimeout';
 
 	protected $endpoints = [
 		'articles' => 'posts',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "articles" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)